js-basics/js-variables-and-numbers_simple-calculator/js/index.js

```diff
@@ -1,7 +1,7 @@
 console.clear();
 
-const operand1 = 12;
-const operand2 = 4;
+let operand1 = 12;
+let operand2 = 4;
```

Comment:
no need to change operand from `const` to `let` because you're not changing `operand2`